packages/ui/src/components/ReviewArea.tsx

```diff
@@ -219,6 +220,34 @@ export default React.memo(function ReviewArea({
       ? currentItem?.colorPalette ?? items[items.length - 1].colorPalette
       : colors.palettes.red;
 
+  const onMemoizedMark = useCallback(
```

Comment:
Do you think it would be cleaner to move these changes into `ReviewButtonBar` so that all the "button knowledge" is in one place? I know that I suggested `ReviewArea` initially, but now that I look at this, I see that this code makes implicit assumptions about the layout and contents of the button bar, which seems a bit brittle. If it's messier the other way, though, no worries.

Reply:
Yeah, I think that makes more sense as well. Just did it!